### ISSUE TITLE
Still getting blocked

### ISSUE THREAD

**Issue by Marco208** [mgp25/Chat-API#1887]
Make sure these boxes are checked before submitting your issue -- thank you!

- [x] You are using PHP > 5.6
- [x] You have installed all extensions and modules listed in [here](https://github.com/mgp25/Chat-API/wiki/Dependencies). PHP Protobuf and Curve25519 are required!
- [x] You have checked if someone has already asked the same issue you have.
- [x] You have read the [wiki](https://github.com/mgp25/Chat-API/wiki).
- [x] Is not a programming question.
- [x] You are using latest Chat API code.

### Error
Post your error below:
Getting blocked. I tried your fix. What is wrong here? This worked for years.

### Debug log
Post your debug log below:

```xml
start
PHP Warning:  Creating default object from empty value in /root/whatsapp-daemon/WhatsAPI-Official/src/custom/functions.php on line 121
Number "316xxxx@s.whatsapp.net" is registered as user "marco" now with level 4095
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1338653109@g.us" is registered as user "xxx" now with level 0
Number "316xxxx-1389726022@g.us" is registered as user "nagwhat" now with level 0
Number "316xxxx-1403882154@g.us" is registered as user "xxxchat" now with level 0
Number "316xxxx-1421244746@g.us" is registered as user "nagwhattwo" now with level 0
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1424604789@g.us" is registered as user "nagwhathome" now with level 0
[==] logging in as 'xxx-Whapp' (316xxxx)
[==] Connecting to WhatsApp

<p>WooHoo!, Phone number 316xxxx connected successfully!</p>tx  <stream:features></stream:features>

tx  <auth user="316xxxx" mechanism="WAUTH-2">2bacc93833313634393739333338372e0145098289e0908804fd07b85eade3944f8e8331343632333038343539</auth>

rx  <from s.whatsapp.net=""></from>

rx  <stream:features></stream:features>

rx  <challenge>e84b5ad6592c4e6def2d9a47108ffd0ef81dd4f9</challenge>

tx  <response>31c772e4a5e06188308f2189d08d8104d754e19a4736a88f0c71880821c03bb66f08eaa3aca72a7d3918c72132e28fd8aa38b8afd7dae911db51847bfd8b7cac66146aaa34aa55df5b77d7a7af</response>

rx  <success t="1462308459" props="5" kind="free" status="active" creation="1462290106" expiration="4444444444">a0998e1ebf4d9ebc2ce3cee5a8e252c2c9ce96f7</success>

tx  <presence name="xxx-Whapp" type="available"></presence>

tx  <iq id="01" xmlns="jabber:iq:privacy" type="get">
tx    <query>
tx      <list name="default"></list>
tx    </query>
tx  </iq>

tx  <iq id="02" xmlns="urn:xmpp:whatsapp:push" type="get" to="s.whatsapp.net">
tx    <config></config>
tx  </iq>

[++] Connected to WhatsApp

rx  <ib from="s.whatsapp.net">
rx    <offline count="0"></offline>
rx  </ib>

rx  <presence from="316xxxx@s.whatsapp.net"></presence>

rx  <iq from="316xxxx@s.whatsapp.net" type="error" id="1">
rx    <error code="404" text="item-not-found"></error>
rx  </iq>

rx  <iq from="s.whatsapp.net" type="result" id="02">
rx    <config platform="gcm" id="1Jo7b_GRHno:APA91biCIRkaFxPVIXoMCsJ5eWZi1bovaoY34OUoK7UIHkejvEYHJjkyEwzGPKTL3fke4r_1jK78-uFY3rFXrtrGF_FTR7ZtRpU_XnTlLO0Td38_7_j7MCe1X4mnEK3LapHQpbRgocOk"></config>
rx  </iq>

Debug timer: 0
Debug timer: 1
Debug timer: 2
Debug timer: 3
Debug timer: 4
Debug timer: 5
Debug timer: 6
Debug timer: 7
Debug timer: 8
Debug timer: 9
Debug timer: 10
Debug timer: 11
Debug timer: 12
Debug timer: 13
Debug timer: 14
Debug timer: 15
Debug timer: 16
Debug timer: 17
Debug timer: 18
Debug timer: 19
Debug timer: 20
Debug timer: 21
Debug timer: 22
Debug timer: 23
Debug timer: 24
Debug timer: 25
Debug timer: 26
Debug timer: 27
tx  <iq id="03" xmlns="w:p" type="get" to="s.whatsapp.net">
tx    <ping></ping>
tx  </iq>

Debug timer: 28
rx  <iq from="s.whatsapp.net" type="result" id="03" t="1462308520"></iq>

Debug timer: 29
Debug timer: 30
Debug timer: 31
Debug timer: 32
Debug timer: 33
Debug timer: 34
Debug timer: 35
Debug timer: 36
Debug timer: 37
Debug timer: 38
Debug timer: 39
Debug timer: 40
Debug timer: 41
Debug timer: 42
Debug timer: 43
Debug timer: 44
Debug timer: 45
Debug timer: 46
Debug timer: 47
Debug timer: 48
Debug timer: 49
tx  <iq id="04" xmlns="w:p" type="get" to="s.whatsapp.net">
tx    <ping></ping>
tx  </iq>

tx  <presence type="active"></presence>

Debug timer: 50
rx  <iq from="s.whatsapp.net" type="result" id="04" t="1462308562"></iq>

Debug timer: 51
Debug timer: 52
Debug timer: 53
Debug timer: 54
Debug timer: 55
Debug timer: 56
Debug timer: 57
Debug timer: 58
Debug timer: 59
Debug timer: 60
Debug timer: 61
Debug timer: 62
Debug timer: 63
Debug timer: 64
Debug timer: 65
Debug timer: 66
Debug timer: 67
Debug timer: 68
Debug timer: 69
Debug timer: 70
Debug timer: 71
Debug timer: 72
Debug timer: 73
Debug timer: 74
Debug timer: 75
Debug timer: 76
Debug timer: 77
Debug timer: 78
Debug timer: 79
Debug timer: 80
Debug timer: 81
tx  <iq id="05" xmlns="w:p" type="get" to="s.whatsapp.net">
tx    <ping></ping>
tx  </iq>

Debug timer: 82
rx  <iq from="s.whatsapp.net" type="result" id="05" t="1462308624"></iq>

Debug timer: 83
Debug timer: 84
Debug timer: 85
Debug timer: 86
Debug timer: 87
Debug timer: 88
Debug timer: 89
Debug timer: 90
Debug timer: 91
Debug timer: 92
Debug timer: 93
Debug timer: 94
Debug timer: 95
Debug timer: 96
Debug timer: 97
Debug timer: 98
Debug timer: 99
tx  <iq id="06" xmlns="w:p" type="get" to="s.whatsapp.net">
tx    <ping></ping>
tx  </iq>

tx  <presence type="active"></presence>

Debug timer: 100
rx  <iq from="s.whatsapp.net" type="result" id="06" t="1462308658"></iq>

Debug timer: 101
Debug timer: 102
Debug timer: 103
Debug timer: 104
Debug timer: 105
Debug timer: 106
Debug timer: 107
Debug timer: 108
Debug timer: 109
Debug timer: 110
Debug timer: 111
Debug timer: 112
Debug timer: 113
Debug timer: 114
Debug timer: 115
Debug timer: 116
Debug timer: 117
Debug timer: 118
Debug timer: 119
Debug timer: 120
Debug timer: 121
Debug timer: 122
Debug timer: 123
Debug timer: 124
Debug timer: 125
Debug timer: 126
Debug timer: 127
Debug timer: 128
Debug timer: 129
Debug timer: 130
tx  <iq id="07" xmlns="w:p" type="get" to="s.whatsapp.net">
tx    <ping></ping>
tx  </iq>

Debug timer: 131
rx  <iq from="s.whatsapp.net" type="result" id="07" t="1462308720"></iq>

Debug timer: 132
Debug timer: 133
Debug timer: 134
Debug timer: 135
Debug timer: 136
Debug timer: 137
Debug timer: 138
Debug timer: 139
Debug timer: 140
Debug timer: 141
Debug timer: 142
Debug timer: 143
Debug timer: 144
Debug timer: 145
Debug timer: 146
Debug timer: 147
Debug timer: 148
start
PHP Warning:  Creating default object from empty value in /root/whatsapp-daemon/WhatsAPI-Official/src/custom/functions.php on line 121
Number "316xxxx@s.whatsapp.net" is registered as user "marco" now with level 4095
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1338653109@g.us" is registered as user "xxx" now with level 0
Number "316xxxx-1389726022@g.us" is registered as user "nagwhat" now with level 0
Number "316xxxx-1403882154@g.us" is registered as user "xxxchat" now with level 0
Number "316xxxx-1421244746@g.us" is registered as user "nagwhattwo" now with level 0
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1424604789@g.us" is registered as user "nagwhathome" now with level 0
[==] logging in as 'xxx-Whapp' (316xxxx)
[==] Connecting to WhatsApp

<p>WooHoo!, Phone number 316xxxx connected successfully!</p>tx  <stream:features></stream:features>

tx  <auth user="316xxxx" mechanism="WAUTH-2">554565453331363439373933333837a0998e1ebf4d9ebc2ce3cee5a8e252c2c9ce96f731343632333038373634</auth>

rx  <from s.whatsapp.net=""></from>

rx  <stream:features></stream:features>

rx  <challenge>88c180cde0319e21d0a2198d366c1a242683b462</challenge>

tx  <response>c48d2feb734effc54148bfeb35d4b113ba48c256aaae03a2d021a85b32ca2f2ab60f1607cb74d044c1635726eeb85bee42460566c2f8fed7f30f7dd5c8f609e04601c03bfa0606c34ef6593b2e</response>

rx  <success t="1462308764" props="5" kind="free" status="active" creation="1462290106" expiration="4444444444">673ca20969b58a46efcfe851aca3393acda1b788</success>

tx  <presence name="xxx-Whapp" type="available"></presence>

tx  <iq id="01" xmlns="jabber:iq:privacy" type="get">
tx    <query>
tx      <list name="default"></list>
tx    </query>
tx  </iq>

tx  <iq id="02" xmlns="urn:xmpp:whatsapp:push" type="get" to="s.whatsapp.net">
tx    <config></config>
tx  </iq>

[++] Connected to WhatsApp

rx  <ib from="s.whatsapp.net">
rx    <offline count="0"></offline>
rx  </ib>

rx  <presence from="316xxxx@s.whatsapp.net"></presence>

rx  <iq from="316xxxx@s.whatsapp.net" type="error" id="1">
rx    <error code="404" text="item-not-found"></error>
rx  </iq>

rx  <iq from="s.whatsapp.net" type="result" id="02">
rx    <config platform="gcm" id="1Jo7b_GRHno:APA91biCIRkaFxPVIXoMCsJ5eWZi1bovaoY34OUoK7UIHkejvEYHJjkyEwzGPKTL3fke4r_1jK78-uFY3rFXrtrGF_FTR7ZtRpU_XnTlLO0Td38_7_j7MCe1X4mnEK3LapHQpbRgocOk"></config>
rx  </iq>

Debug timer: 0
Debug timer: 1
Debug timer: 2
Debug timer: 3
Debug timer: 4
Debug timer: 5
Debug timer: 6
Debug timer: 7
Debug timer: 8
Debug timer: 9
Debug timer: 10
Debug timer: 11
Debug timer: 12
Debug timer: 13
Debug timer: 14
Debug timer: 15
Debug timer: 16
Debug timer: 17
Debug timer: 18
Debug timer: 19
Debug timer: 20
Debug timer: 21
Debug timer: 22
Debug timer: 23
Debug timer: 24
Debug timer: 25
Debug timer: 26
Debug timer: 27
tx  <iq id="03" xmlns="w:p" type="get" to="s.whatsapp.net">
tx    <ping></ping>
tx  </iq>

Debug timer: 28
rx  <iq from="s.whatsapp.net" type="result" id="03" t="1462308825"></iq>

Debug timer: 29
Debug timer: 30
Debug timer: 31
Debug timer: 32
Debug timer: 33
Debug timer: 34
Debug timer: 35
Debug timer: 36
Debug timer: 37
Debug timer: 38
Debug timer: 39
Debug timer: 40
Debug timer: 41
Debug timer: 42
Debug timer: 43
Debug timer: 44
Debug timer: 45
Debug timer: 46
Debug timer: 47
Debug timer: 48
Debug timer: 49
tx  <iq id="04" xmlns="w:p" type="get" to="s.whatsapp.net">
tx    <ping></ping>
tx  </iq>

tx  <presence type="active"></presence>

Debug timer: 50
rx  <iq from="s.whatsapp.net" type="result" id="04" t="1462308868"></iq>

Debug timer: 51
Debug timer: 52
Debug timer: 53
Debug timer: 54
Debug timer: 55
Debug timer: 56
Debug timer: 57
Debug timer: 58
Debug timer: 59
Debug timer: 60
Debug timer: 61
Debug timer: 62
Debug timer: 63
Debug timer: 64
Debug timer: 65
Debug timer: 66
Debug timer: 67
Debug timer: 68
Debug timer: 69
Debug timer: 70
Debug timer: 71
Debug timer: 72
Debug timer: 73
Debug timer: 74
Debug timer: 75
Debug timer: 76
Debug timer: 77
Debug timer: 78
Debug timer: 79
Debug timer: 80
Debug timer: 81
tx  <iq id="05" xmlns="w:p" type="get" to="s.whatsapp.net">
tx    <ping></ping>
tx  </iq>

Debug timer: 82
rx  <iq from="s.whatsapp.net" type="result" id="05" t="1462308930"></iq>

Debug timer: 83
Debug timer: 84
Debug timer: 85
Debug timer: 86
Debug timer: 87
Debug timer: 88
Debug timer: 89
Debug timer: 90
Debug timer: 91
Debug timer: 92
Debug timer: 93
Debug timer: 94
Debug timer: 95
Debug timer: 96
Debug timer: 97
Debug timer: 98
Debug timer: 99
tx  <iq id="06" xmlns="w:p" type="get" to="s.whatsapp.net">
tx    <ping></ping>
tx  </iq>

tx  <presence type="active"></presence>

Debug timer: 100
rx  <iq from="s.whatsapp.net" type="result" id="06" t="1462308966"></iq>

Debug timer: 101
Debug timer: 102
Debug timer: 103
Debug timer: 104
Debug timer: 105
Debug timer: 106
Debug timer: 107
Debug timer: 108
Debug timer: 109
Debug timer: 110
Debug timer: 111
Debug timer: 112
Debug timer: 113
Debug timer: 114
Debug timer: 115
Debug timer: 116
Debug timer: 117
Debug timer: 118
Debug timer: 119
Debug timer: 120
rx  <stream:error>
rx    <system-shutdown></system-shutdown>
rx  </stream:error>

<p>Booo!, Phone number 316xxxx is disconnected!</p>Debug timer: 121
PHP Fatal error:  Uncaught exception 'ConnectionException' with message 'Connection Closed!' in /root/whatsapp-daemon/WhatsAPI-Official/src/whatsprot.class.php:299
Stack trace:
#0 /root/whatsapp-daemon/WhatsAPI-Official/src/whatsapp-daemon.php(66): WhatsProt->pollMessage()
#1 {main}
  thrown in /root/whatsapp-daemon/WhatsAPI-Official/src/whatsprot.class.php on line 299
start
PHP Warning:  Creating default object from empty value in /root/whatsapp-daemon/WhatsAPI-Official/src/custom/functions.php on line 121
Number "316xxxx@s.whatsapp.net" is registered as user "marco" now with level 4095
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1338653109@g.us" is registered as user "xxx" now with level 0
Number "316xxxx-1389726022@g.us" is registered as user "nagwhat" now with level 0
Number "316xxxx-1403882154@g.us" is registered as user "xxxchat" now with level 0
Number "316xxxx-1421244746@g.us" is registered as user "nagwhattwo" now with level 0
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1424604789@g.us" is registered as user "nagwhathome" now with level 0
[==] logging in as 'xxx-Whapp' (316xxxx)
[==] Connecting to WhatsApp

<p>WooHoo!, Phone number 316xxxx connected successfully!</p>tx  <stream:features></stream:features>

tx  <auth user="316xxxx" mechanism="WAUTH-2">c8c575b63331363439373933333837673ca20969b58a46efcfe851aca3393acda1b78831343632333039303035</auth>

rx  <from s.whatsapp.net=""></from>

rx  <stream:features></stream:features>

rx  <challenge>7380badf0d78b0891cf9c8b9762d1f8dd4838464</challenge>

tx  <response>936c8eeb5ada9e8568c97f3315de75a407f7affb3e505c16d93b3c6496538a58fbe7ea854d2e1fe0875cbe82734ae8e4366cc3c52f73dddc5a8bddf8d246476740ebfc27ed93e4532a4a3b2a7b</response>

rx  <success t="1462309005" props="5" kind="free" status="active" creation="1462290106" expiration="4444444444">ee22728689c280572376ac4a704b3c48cfe6249b</success>

tx  <presence name="xxx-Whapp" type="available"></presence>

tx  <iq id="01" xmlns="jabber:iq:privacy" type="get">
tx    <query>
tx      <list name="default"></list>
tx    </query>
tx  </iq>

tx  <iq id="02" xmlns="urn:xmpp:whatsapp:push" type="get" to="s.whatsapp.net">
tx    <config></config>
tx  </iq>

[++] Connected to WhatsApp

rx  <ib from="s.whatsapp.net">
rx    <offline count="0"></offline>
rx  </ib>

rx  <presence from="316xxxx@s.whatsapp.net"></presence>

rx  <iq from="316xxxx@s.whatsapp.net" type="error" id="1">
rx    <error code="404" text="item-not-found"></error>
rx  </iq>

rx  <iq from="s.whatsapp.net" type="result" id="02">
rx    <config platform="gcm" id="1Jo7b_GRHno:APA91biCIRkaFxPVIXoMCsJ5eWZi1bovaoY34OUoK7UIHkejvEYHJjkyEwzGPKTL3fke4r_1jK78-uFY3rFXrtrGF_FTR7ZtRpU_XnTlLO0Td38_7_j7MCe1X4mnEK3LapHQpbRgocOk"></config>
rx  </iq>

Debug timer: 0
Debug timer: 1
Debug timer: 2
Debug timer: 3
Debug timer: 4
Debug timer: 5
Debug timer: 6
Debug timer: 7
Debug timer: 8
Debug timer: 9
Debug timer: 10
Debug timer: 11
Debug timer: 12
Debug timer: 13
Debug timer: 14
Debug timer: 15
Debug timer: 16
Debug timer: 17
Debug timer: 18
Debug timer: 19
Debug timer: 20
Debug timer: 21
Debug timer: 22
Debug timer: 23
Debug timer: 24
Debug timer: 25
Debug timer: 26
Debug timer: 27
Debug timer: 28
tx  <iq id="03" xmlns="w:p" type="get" to="s.whatsapp.net">
tx    <ping></ping>
tx  </iq>

Debug timer: 29
rx  <iq from="s.whatsapp.net" type="result" id="03" t="1462309068"></iq>

Debug timer: 30
Debug timer: 31
Debug timer: 32
Debug timer: 33
Debug timer: 34
Debug timer: 35
Debug timer: 36
Debug timer: 37
Debug timer: 38
Debug timer: 39
Debug timer: 40
Debug timer: 41
Debug timer: 42
rx  <stream:error>
rx    <system-shutdown></system-shutdown>
rx  </stream:error>

<p>Booo!, Phone number 316xxxx is disconnected!</p>Debug timer: 43
PHP Fatal error:  Uncaught exception 'ConnectionException' with message 'Connection Closed!' in /root/whatsapp-daemon/WhatsAPI-Official/src/whatsprot.class.php:299
Stack trace:
#0 /root/whatsapp-daemon/WhatsAPI-Official/src/whatsapp-daemon.php(66): WhatsProt->pollMessage()
#1 {main}
  thrown in /root/whatsapp-daemon/WhatsAPI-Official/src/whatsprot.class.php on line 299
start
PHP Warning:  Creating default object from empty value in /root/whatsapp-daemon/WhatsAPI-Official/src/custom/functions.php on line 121
Number "316xxxx@s.whatsapp.net" is registered as user "marco" now with level 4095
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1338653109@g.us" is registered as user "xxx" now with level 0
Number "316xxxx-1389726022@g.us" is registered as user "nagwhat" now with level 0
Number "316xxxx-1403882154@g.us" is registered as user "xxxchat" now with level 0
Number "316xxxx-1421244746@g.us" is registered as user "nagwhattwo" now with level 0
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1424604789@g.us" is registered as user "nagwhathome" now with level 0
[==] logging in as 'xxx-Whapp' (316xxxx)
[==] Connecting to WhatsApp

<p>WooHoo!, Phone number 316xxxx connected successfully!</p>tx  <stream:features></stream:features>

tx  <auth user="316xxxx" mechanism="WAUTH-2">b102f7ad3331363439373933333837ee22728689c280572376ac4a704b3c48cfe6249b31343632333039303933</auth>

rx  <from s.whatsapp.net=""></from>

rx  <stream:features></stream:features>

rx  <challenge>2f471b3866b192da32925b02d096f72c9ab9b593</challenge>

tx  <response>754b58fbeddc64764e35dba8d3b852c7fdcd5a45f585d2bad944fd30a89a85b3f4027b70380badafd243d5d6a8b96941656a8acbe02c947db0df26f8a481cc12961ebedec9af89c93fcc5b0613</response>

rx  <success t="1462309093" props="5" kind="free" status="active" creation="1462290106" expiration="4444444444">d8e49f794788b6b5144477bb07acc716ad0e0128</success>

tx  <presence name="xxx-Whapp" type="available"></presence>

tx  <iq id="01" xmlns="jabber:iq:privacy" type="get">
tx    <query>
tx      <list name="default"></list>
tx    </query>
tx  </iq>

tx  <iq id="02" xmlns="urn:xmpp:whatsapp:push" type="get" to="s.whatsapp.net">
tx    <config></config>
tx  </iq>

[++] Connected to WhatsApp

rx  <ib from="s.whatsapp.net">
rx    <offline count="0"></offline>
rx  </ib>

rx  <presence from="316xxxx@s.whatsapp.net"></presence>

rx  <iq from="316xxxx@s.whatsapp.net" type="error" id="1">
rx    <error code="404" text="item-not-found"></error>
rx  </iq>

rx  <iq from="s.whatsapp.net" type="result" id="02">
rx    <config platform="gcm" id="1Jo7b_GRHno:APA91biCIRkaFxPVIXoMCsJ5eWZi1bovaoY34OUoK7UIHkejvEYHJjkyEwzGPKTL3fke4r_1jK78-uFY3rFXrtrGF_FTR7ZtRpU_XnTlLO0Td38_7_j7MCe1X4mnEK3LapHQpbRgocOk"></config>
rx  </iq>

Debug timer: 0
Debug timer: 1
Debug timer: 2
Debug timer: 3
Debug timer: 4
Debug timer: 5
Debug timer: 6
Debug timer: 7
Debug timer: 8
Debug timer: 9
Debug timer: 10
Debug timer: 11
Debug timer: 12
Debug timer: 13
Debug timer: 14
Debug timer: 15
Debug timer: 16
Debug timer: 17
Debug timer: 18
Debug timer: 19
Debug timer: 20
rx  <stream:error>
rx    <system-shutdown></system-shutdown>
rx  </stream:error>

<p>Booo!, Phone number 316xxxx is disconnected!</p>Debug timer: 21
PHP Fatal error:  Uncaught exception 'ConnectionException' with message 'Connection Closed!' in /root/whatsapp-daemon/WhatsAPI-Official/src/whatsprot.class.php:299
Stack trace:
#0 /root/whatsapp-daemon/WhatsAPI-Official/src/whatsapp-daemon.php(66): WhatsProt->pollMessage()
#1 {main}
  thrown in /root/whatsapp-daemon/WhatsAPI-Official/src/whatsprot.class.php on line 299
start
PHP Warning:  Creating default object from empty value in /root/whatsapp-daemon/WhatsAPI-Official/src/custom/functions.php on line 121
Number "316xxxx@s.whatsapp.net" is registered as user "marco" now with level 4095
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1338653109@g.us" is registered as user "xxx" now with level 0
Number "316xxxx-1389726022@g.us" is registered as user "nagwhat" now with level 0
Number "316xxxx-1403882154@g.us" is registered as user "xxxchat" now with level 0
Number "316xxxx-1421244746@g.us" is registered as user "nagwhattwo" now with level 0
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1424604789@g.us" is registered as user "nagwhathome" now with level 0
[==] logging in as 'xxx-Whapp' (316xxxx)
[==] Connecting to WhatsApp

<p>WooHoo!, Phone number 316xxxx connected successfully!</p>tx  <stream:features></stream:features>

tx  <auth user="316xxxx" mechanism="WAUTH-2">c7da93913331363439373933333837d8e49f794788b6b5144477bb07acc716ad0e012831343632333039313338</auth>

rx  <from s.whatsapp.net=""></from>

rx  <stream:features></stream:features>

rx  <challenge>091aeee92166b4b3428d02a0dff63d1a6b52fcb5</challenge>

tx  <response>9b5ef1266b56e616902156680649a3dd33c1a52a49c1dece47259f276e57ce9b2b1f97388c82ecdb4388e8b8d1f6835ada25a18766b31a9509630af24e1362e6ebc3ac70d5b87da6a6e0ee9e11</response>

rx  <failure>
rx    <not-authorized></not-authorized>
rx  </failure>

PHP Fatal error:  Uncaught exception 'LoginFailureException' in /root/whatsapp-daemon/WhatsAPI-Official/src/Login.php:54
Stack trace:
#0 /root/whatsapp-daemon/WhatsAPI-Official/src/whatsprot.class.php(286): Login->doLogin()
#1 /root/whatsapp-daemon/WhatsAPI-Official/src/whatsapp-daemon.php(42): WhatsProt->loginWithPassword('AypidxrZVpq/9tz...')
#2 {main}
  thrown in /root/whatsapp-daemon/WhatsAPI-Official/src/Login.php on line 54
start
PHP Warning:  Creating default object from empty value in /root/whatsapp-daemon/WhatsAPI-Official/src/custom/functions.php on line 121
Number "316xxxx@s.whatsapp.net" is registered as user "marco" now with level 4095
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1338653109@g.us" is registered as user "xxx" now with level 0
Number "316xxxx-1389726022@g.us" is registered as user "nagwhat" now with level 0
Number "316xxxx-1403882154@g.us" is registered as user "xxxchat" now with level 0
Number "316xxxx-1421244746@g.us" is registered as user "nagwhattwo" now with level 0
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1424604789@g.us" is registered as user "nagwhathome" now with level 0
[==] logging in as 'xxx-Whapp' (316xxxx)
[==] Connecting to WhatsApp

<p>WooHoo!, Phone number 316xxxx connected successfully!</p>tx  <stream:features></stream:features>

tx  <auth user="316xxxx" mechanism="WAUTH-2">c7da93913331363439373933333837d8e49f794788b6b5144477bb07acc716ad0e012831343632333039313339</auth>

rx  <from s.whatsapp.net=""></from>

rx  <stream:features></stream:features>

rx  <challenge>05f2f18c78d5b5f26a5e250c491313d6b654197d</challenge>

tx  <response>bbc79c8f0b36405cf2f27f52bc985baf450084bf8f0c3a36269e0b51a3c701840ecaafee59cd38ca654126bfa1203f0f84e516520679e83876405917006814318608c0ae042e5f48625649b06a</response>

rx  <failure>
rx    <not-authorized></not-authorized>
rx  </failure>

PHP Fatal error:  Uncaught exception 'LoginFailureException' in /root/whatsapp-daemon/WhatsAPI-Official/src/Login.php:54
Stack trace:
#0 /root/whatsapp-daemon/WhatsAPI-Official/src/whatsprot.class.php(286): Login->doLogin()
#1 /root/whatsapp-daemon/WhatsAPI-Official/src/whatsapp-daemon.php(42): WhatsProt->loginWithPassword('AypidxrZVpq/9tz...')
#2 {main}
  thrown in /root/whatsapp-daemon/WhatsAPI-Official/src/Login.php on line 54
start
PHP Warning:  Creating default object from empty value in /root/whatsapp-daemon/WhatsAPI-Official/src/custom/functions.php on line 121
Number "316xxxx@s.whatsapp.net" is registered as user "marco" now with level 4095
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1338653109@g.us" is registered as user "xxx" now with level 0
Number "316xxxx-1389726022@g.us" is registered as user "nagwhat" now with level 0
Number "316xxxx-1403882154@g.us" is registered as user "xxxchat" now with level 0
Number "316xxxx-1421244746@g.us" is registered as user "nagwhattwo" now with level 0
Number "316xxxx@s.whatsapp.net" is registered as user "xxx" now with level 4095
Number "316xxxx-1424604789@g.us" is registered as user "nagwhathome" now with level 0
[==] logging in as 'xxx-Whapp' (316xxxx)
[==] Connecting to WhatsApp

<p>WooHoo!, Phone number 316xxxx connected successfully!</p>tx  <stream:features></stream:features>

tx  <auth user="316xxxx" mechanism="WAUTH-2">c7da93913331363439373933333837d8e49f794788b6b5144477bb07acc716ad0e012831343632333039313432</auth>

rx  <from s.whatsapp.net=""></from>

rx  <stream:features></stream:features>

rx  <challenge>b09fcb78b3d3fdd7c98fc499ebcdeb05affa0817</challenge>

tx  <response>c9cb5a25ce5a5c478718923734fedb49973239b89706a890e7b681ded82395c24b5e80abd3df23dfeb9534394158d7b3207e196e9a44cf32c4233bb97791c165739e6c29e4a697b52895e15cba</response>

rx  <failure>
rx    <not-authorized></not-authorized>
rx  </failure>

PHP Fatal error:  Uncaught exception 'LoginFailureException' in /root/whatsapp-daemon/WhatsAPI-Official/src/Login.php:54
Stack trace:
#0 /root/whatsapp-daemon/WhatsAPI-Official/src/whatsprot.class.php(286): Login->doLogin()
#1 /root/whatsapp-daemon/WhatsAPI-Official/src/whatsapp-daemon.php(42): WhatsProt->loginWithPassword('AypidxrZVpq/9tz...')
#2 {main}
  thrown in /root/whatsapp-daemon/WhatsAPI-Official/src/Login.php on line 54
```

**Comment by mgp25** [comment]
@Marco208 please read this comment: https://github.com/mgp25/Chat-API/issues/1872#issuecomment-215912828

i told you already the solution, dont open duplicate issues. Only 2 pings!

**Comment by pereira49** [comment]
@mgp25 i implemented the solution #1872 (.only 2 pings). I registered number, no send/receive any message, not sync because, not receive any message, so my contact list is empty
[log_whats.txt](https://github.com/mgp25/Chat-API/files/250935/log_whats.txt)

**Comment by mgp25** [comment]
@pereira49 See you debug log again, its bad coded, please check your code, because is not API issue, check the id

Regards
